.github/workflows/swift-toolchain.yml

```diff
@@ -1987,15 +1970,55 @@ jobs:
           $RTLPath = cygpath -w ${{ github.workspace }}/BinaryCache/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/bin
           echo ${RTLPath} | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
 
-      - name: Extract swift-syntax and vfs-overlay
+      - name: Extract swift-syntax
         run: |
           $module = "${{ github.workspace }}/BinaryCache/swift-syntax/cmake/modules/SwiftSyntaxConfig.cmake"
           $bindir = cygpath -m ${{ github.workspace }}/BinaryCache/swift-syntax
           (Get-Content $module).Replace('<BINARY_DIR>', "${bindir}") | Set-Content $module
 
+      - name: Create vfs-overlay
+        run: |
           $VfsOverlay = "${{ github.workspace }}/BinaryCache/swift/stdlib/windows-vfs-overlay.yaml"
           $ModuleMapDir = cygpath -m ${{ github.workspace }}/BinaryCache/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/share
-          (Get-Content $VfsOverlay).Replace("<MODULE_MAP_DIR>", "${ModuleMapDir}") | Set-Content $VfsOverlay
+          $Win10SdkRoot = Get-ItemPropertyValue `
+            -Path "Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Kits\Installed Roots" `
+            -Name "KitsRoot10"
+          $Win10SdkRoot = cygpath -m ${Win10SdkRoot}
+          $InstallerLocation = Join-Path "${env:ProgramFiles(x86)}" "Microsoft Visual Studio" "Installer"
```

Comment:
I think that [IO.Path]::Combine is slightly better, but let’s get @jeffdav’s take on it.

Reply:
This works with PowerShell Core (pwsh), which is what we use on CI.